src/pages/ErrorPage/SessionExpiredPage.tsx

```diff
@@ -39,7 +39,7 @@ function SessionExpiredPage() {
                                     Navigation.goBack();
                                     return;
                                 }
-                                HybridAppModule.closeReactNativeApp({shouldSignOut: true, shouldSetNVP: false});
+                                closeReactNativeApp({shouldSignOut: true, shouldSetNVP: false});
```

Comment:
Ok, so now we have a new `closeReactNativeApp` function which handles root status bar being enabled or not 🤔 Let's look into this again, there might be a better way